handbook/security.md

```diff
@@ -255,14 +256,135 @@ We configure Chrome on company-owned devices with a basic policy.
+| Allow force pushes                                               | 🚫     | We do not have a need this, so we do not allow it.                                                             |
+| Allow deletions                                                  | 🚫     | We do not want ANYONE to be able to delete the *main* branch.                                                         |
+
+### Scanning tools
+Though not technically a part of GitHub itself, we feel like the security tools we use to scan our code, workflows, and GitHub configuration are part of our overall GitHub configuration.
+
+#### SAST and configuration scanning
+| Scanning Tool                                       | Purpose                                                                                                                                              | Configuration                                                                                                  |
+| --------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| [OSSF Scorecard](https://github.com/ossf/scorecard) | Scan our GitHub repository for best practices and send problems to GitHub Security.                                                                  | [scorecard-analysis.yml](https://github.com/fleetdm/fleet/blob/main/.github/workflows/scorecards-analysis.yml) |
+| [CodeQL](https://codeql.github.com/)                | Discover vulnerabilities across our codebase, both in the backend and frontend code.                                                                 | [codeql-analysis.yml](https://github.com/fleetdm/fleet/blob/main/.github/workflows/codeql-analysis.yml)        |
+| [gosec](https://github.com/securego/gosec)          | Scan golang code for common security mistakes. We use gosec as one of the linters used by [golangci-lint](https://github.com/golangci/golangci-lint) | [golangci-lint.yml](https://github.com/fleetdm/fleet/blob/main/.github/workflows/golangci-lint.yml)            |
+
+We are planning on adding [tfsec](https://github.com/aquasecurity/tfsec) to scan for configuration vulnerabilities in the Terraform code provided to deploy Fleet infrastructure in the cloud. 
+Once we have full coverage from a static analysis point of view, we will evaluate dynamic analysis
+and fuzzing options.
+
+#### Dependabot
+As described in *Code security and analysis*, we use Dependabot for security updates to libraries.
+Our [dependabot.yml](https://github.com/fleetdm/fleet/blob/main/.github/dependabot.yml) only
+mentions GitHub actions. Through the configuration of the repository, security update to
```

Comment:
Please reword "Through the configuration of the repository, security update to dependencies generate pull requests for everything in the repository."

This is a sentence fragment. Is it telling you to run the security update? There seem to be some keywords missing. Should "update" be plural? These are some guesses as to what went wrong.

Reply:
@GuillaumeRoss  Your turn!